packages/block-editor/src/store/actions.js

```diff
@@ -523,8 +523,28 @@ export function* insertBlocks(
 }
 
 /**
- * Returns an action object used in signalling that the insertion point should
- * be shown.
+ * Sets the insertion point without showing it to users.
+ *
+ * Components like <Inserter> will default to inserting blocks at this point.
+ *
+ * @param {?string} rootClientId Optional root client ID of block list on
+ *                               which to insert.
+ * @param {?number} index        Index at which block should be inserted.
```

Comment:
I'm thinking that these shouldn't be documented as optional, as they have the effect of unsetting the `rootClientId` and `index` in the store (and I'm not sure they should). There's the same issue with `showInsertionPoint`.

Passing a rootClientId without an index or index without rootClientId doesn't seem like good practice. If unsetting the values is required an additional unsetInsertionPoint action could be introduced.

Reply:
Passing no args is also not done anywhere in the codebase.

Reply:
Turns out `undefined` is used for the top-level block list. I thought this might be one of those places that uses an empty string.

I at least marked index as required.